src/java.base/share/classes/jdk/internal/misc/CDS.java

```diff
@@ -59,4 +76,123 @@
      * Check if sharing is enabled via the UseSharedSpaces flag.
      */
     public static native boolean isSharingEnabled();
+
+    /**
+     * check if -XX:+DumpLoadedClassList and given file is open
+     */
+    public static boolean isDumpLoadedClassList() {
```

Comment:
I agree with Ioi's suggestion to rename this to `isDumpingClassList` which describes what the VM is doing.

Reply:
Done